data/field_weather.s

```diff
@@ -0,0 +1,103 @@
+	.include "asm/macros.inc"
+	.include "constants/constants.inc"
+
+	.section .rodata
+
+	.align 2
+gUnknown_0854014C:: @ 854014C
```

Comment:
Might as well move this into the .c file now.

Reply:
not me, these files need changes, because each pal has only 256 bytes whereas it should have 0x1000 bytes and I am NOT going to do this 😋